.drone.yml

```diff
@@ -0,0 +1,53 @@
+---
+kind: pipeline
+name: setup-and-test
```

Comment:
(nit) I would just name this `test`, I think it's assumed that we'll have things set up to let us test it.